spec/factories/communication_template_factory.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+FactoryGirl.define do
+  factory :communication_template do
+    association :language
+    category 'new_job_template'
```

Comment:
might cause problems since there is a unique constraint for `language <> category`